deploy/wip-catalog/templates/apiserver.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     spec:
       containers:
       - name: apiserver
-        image: {{ .Values.registry }}/apiserver:{{ if .Values.k8sApiServerVersion }}{{ .Values.k8sApiServerVersion }}{{ else }}{{ default "latest" .Values.version }}{{ end }}
+        image: {{ if .Values.registry }}{{ cat .Values.registry "/"}}{{ end }}apiserver:{{ if .Values.k8sApiServerVersion }}{{ .Values.k8sApiServerVersion }}{{ else }}{{ default "latest" .Values.version }}{{ end }}
```

Comment:
It's more straightforward to _not_ use `cat` here, and instead move the `/` (unquoted) to after the `}}`.